src/services/Facts/index.js

```diff
@@ -0,0 +1,24 @@
+import * as FactsFragments from '../graphql/fragments/Facts';
+import * as FactsQueries from '../graphql/queries/Facts';
+import { fetchGqlData } from '../shared';
+
+export const SERVICES = {
+  loadFacts(pipelinekeys, callback) {
+    const gqlEndpoint = 'messages';
+
+    const selectionFragments = `
+      ${FactsFragments.factsFragment}
+    `;
+
+    const query = `
+      ${selectionFragments}
+      ${FactsQueries.FactsQuery}
+    `;
+
+    const variables = {
+      pipelinekeys
```

Comment:
Assuming that event time and keyword tags will also be integrated as `variables`?

Reply:
If we do that, how will the view differ from the activity feed?